### PR TITLE
Replace PHP SDK with `avtocod/specs-php` package

### PR DESCRIPTION
**Pull request by Reallife**
| Q             | A
| ------------- | ---
| Bug fix?      | Yes or No
| New feature?  | Yes or No

### Added

- Composer dependency `avtocod/specs-php`  [#137]

### Removed

- **PHP SDK**. `PHP SDK` was moved to separate repository [#137]
- Issue templates for sdk bugs and feature requests [#137]
- `Makefile` goals `test` and `test-cover` [#137]
- Installing `xdebug` in `Dockerfile` [#137]

[#137]:https://github.com/avtocod/specs/issues/137

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I wrote unit tests for my code _(if tests is required for my changes)_
- [x] I have made changes in `CHANGELOG.md` file

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,23 +17,20 @@
     "require": {
         "php": "^7.2",
         "ext-json": "*",
-        "illuminate/support": "^5.6 || ~6.0 || ~7.0",
-        "ocramius/package-versions": "^1.4",
-        "tarampampam/wrappers-php": "^1.5"
+        "avtocod/specs-php": "dev-fix-tests-and-other-stuff"
```
Comment (tarampampam) [inline composer.json:20]: Do not forget to replace this with something like `^v1.0` =)

File: composer.json [hunk 2]
```diff
@@ -49,6 +46,13 @@
             "@phpunit-cover"
         ]
     },
+    "prefer-stable": true,
+    "repositories":[
+        {
+            "type": "vcs",
+            "url": "git@github.com:Reallife/specs-php.git"
+        }
+    ],
```
Comment (tarampampam) [inline composer.json:55]:
```suggestion
    ],
    "suggest": {
        "avtocod/specs-php": "SDK for a working with specifications data."
    },
```

File: composer.json [hunk 3]
```diff
@@ -49,6 +46,13 @@
             "@phpunit-cover"
         ]
     },
+    "prefer-stable": true,
+    "repositories":[
```
Comment (tarampampam) [inline composer.json:50]: Do not forget to remove =)

File: sdk/php/src/Path.php [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<?php
```
Comment (tarampampam) [inline sdk/php/src/Path.php:1]: Delete `./sdk` completely, dude!

**Review by tarampampam — CHANGES_REQUESTED** [review]

File: sdk/php/src/Path.php (hunk 4 above)
Comment (Reallife) [reply sdk/php/src/Path.php]: Done

File: composer.json (hunk 3 above)
Comment (Reallife) [reply composer.json:23]: Yep, as soon as `avtocod/specs-php` will be released

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
...
 
+## UNRELEASED
+
+### Added
+
+- Composer dependency `avtocod/specs-php`  [#137]
+
+### Removed
+
+- **PHP SDK**. `PHP SDK` was moved to separate repository [#137]
```
Comment (tarampampam) [inline CHANGELOG.md:15]:
```suggestion
- **PHP SDK** - moved to [separate repository `avtocod/specs-php`](https://github.com/avtocod/specs-php) [#137]
```

**Review by tarampampam — CHANGES_REQUESTED** [review]

**Review by tarampampam — APPROVED** [review]

**Pull request merged**